internal/template-validator/webhooks/utils.go

```diff
@@ -69,29 +70,35 @@ func ToAdmissionResponse(causes []metav1.StatusCause) *admissionv1.AdmissionResp
 	}
 }
 
-func GetAdmissionReviewVM(ar *admissionv1.AdmissionReview) (*kubevirt.VirtualMachine, *kubevirt.VirtualMachine, error) {
+func GetAdmissionReviewVM(ar *admissionv1.AdmissionReview) (*kubevirt.VirtualMachine, error) {
 	if ar.Request.Resource.Resource != "virtualmachines" {
-		return nil, nil, fmt.Errorf("expect resource %v to be '%s'", ar.Request.Resource, "virtualmachines")
+		return nil, fmt.Errorf("expect resource %v to be '%s'", ar.Request.Resource, "virtualmachines")
 	}
 
-	var err error
-	raw := ar.Request.Object.Raw
-	newVM := kubevirt.VirtualMachine{}
-
-	err = json.Unmarshal(raw, &newVM)
+	newVM := &kubevirt.VirtualMachine{}
+	err := json.Unmarshal(ar.Request.Object.Raw, newVM)
 	if err != nil {
-		return nil, nil, err
+		return nil, err
 	}
 
-	if ar.Request.Operation == admissionv1.Update {
-		raw := ar.Request.OldObject.Raw
-		oldVM := kubevirt.VirtualMachine{}
-		err = json.Unmarshal(raw, &oldVM)
-		if err != nil {
-			return nil, nil, err
-		}
-		return &newVM, &oldVM, nil
+	return newVM, nil
+}
+
+func GetAdmissionReviewTemplate(ar *admissionv1.AdmissionReview) (*templatev1.Template, error) {
+	const resourceName = "templates"
```

Comment:
can you please export this variable as global constant? You are using it 4 times in this PR.

Reply:
Ok

Reply:
On second thought, I would like to leave it as is. I'm not sure in which file to put the global constant, and I don't want to unnecessarily mix validator and operator code.

Reply:
Fair point